giraffe/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@influxdata/giraffe",
-  "version": "2.15.8",
+  "version": "2.15.9",
```

Comment:
i think we also need to update `giraffe/stories/package.json`'s version number.

@TCL735 is that true?

Reply:
Yes, ideally everyone should update it. But as long as one person (me) remembers to update it periodically, i think it's fine. I can't seem to find where `giraffe/stories/package.json`'s version number is actually displayed other than in that file, ie why it's useful.

Whereas the version number in `giraffe/giraffe/package.json` allows us to publish correctly.

Reply:
Updated the version number in `giraffe/stories/package.json`